telegram/utils/request.py

```diff
@@ -135,12 +135,14 @@ def _parse(json_data):
     except ValueError:
         raise TelegramError('Invalid server response')
 
-    if not data.get('ok') and data.get('description'):
-        return data['description']
+    if not data.get('ok'):
+        if data.get('parameters') and data['parameters'].get('migrate_to_chat_id'):
+            raise ChatMigrated(str(data['parameters']['migrate_to_chat_id']))
+        if data.get('description'):
+            return data['description']
```

Comment:
while you're at it, can you save `data.get('description')` to a variable as well?